pom.xml

```diff
@@ -134,6 +133,7 @@
         <plugin.retirejs.breakOnFailure>false</plugin.retirejs.breakOnFailure>
         <plugin.jetty.version>12.0.19</plugin.jetty.version>
         <plugin.protoc-jar.version>3.11.4</plugin.protoc-jar.version>
+        <plugin.datanucleus.version>5.2.1</plugin.datanucleus.version>
```

Comment:
This should be `6.0.0-release`, [as used by Alpine](https://github.com/stevespringett/Alpine/blob/28c424604b37a093c754e14db02a4092866cf916/pom.xml#L109).